airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -1443,6 +1445,12 @@ def get_job(
         job = client.get_job(job_id=job_id, project=project_id, location=location)
         return job
 
+    @staticmethod
+    def _custom_job_id(configuration: Dict[str, Any]) -> str:
+        hash_base = json.dumps(configuration, sort_keys=True)
+        uniqueness_suffix = hashlib.md5(hash_base.encode()).hexdigest()
+        return f"airflow_{int(time.time())}_{uniqueness_suffix}"
```

Comment:
Is it not safer to use microseconds from epoch than seconds?

```suggestion
        microseconds_from_epoch = int((datetime.now() - datetime.fromtimestamp(0)) / timedelta(microseconds=1))
        return f"airflow_{microseconds_from_epoch}_{uniqueness_suffix}"
```

Reply:
imports required for suggestion above `from datetime import datetime, timedelta`